### PR TITLE
Containerize api-policy-component

### PR DESCRIPTION
**Pull request by BerniVarga**
Tested in xp.

### CODE REVIEW

**Comment by tamas-molnar** [comment]
+1

File: notifications-rw-sidekick@.service [hunk 1]
```diff
@@ -8,6 +8,7 @@ ExecStart=/bin/sh -c "\
   etcdctl set ft/services/notifications-rw/healthcheck true; \
   etcdctl mkdir /ft/services/notifications-rw/servers; \
   etcdctl set /ft/healthcheck/notifications-rw-%i /__health; \
+  etcdctl set /vulcand/frontends/public-services-notifications/frontend '{\"Type\": \"http\", \"BackendId\": \"vcb-notifications-rw\", \"Route\": \"PathRegexp(`/notifications.*`) && Host(`public-services`)\"}'; \
```
Comment (Nesze) [inline notifications-rw-sidekick@.service]: I wonder why is this working, "PathRegexp(`/notifications.*`)".
I thought it should match the whole path.

**Comment by Nesze** [comment]
+1

File: notifications-rw-sidekick@.service (hunk 1 above)
Comment (BerniVarga) [inline notifications-rw-sidekick@.service]: I tried to find the answer in vulcan's code, but couldn't find it. Anyway, I changed the regex path as you suggested - to keep consistency.

**Comment by martingartonft** [comment]
+1

**Pull request merged**